src/modules/curations/itemCuration/reducer.ts

```diff
@@ -43,7 +60,47 @@ export function itemCurationReducer(state: ItemCurationState = INITIAL_STATE, ac
         },
         error: null
       }
+    }
+
+    case PUBLISH_THIRD_PARTY_ITEMS_SUCCESS: {
+      const { itemCurations } = action.payload
+      const oldItemCurations = state.data[action.payload.collectionId]
+      const mergedItemCurations = oldItemCurations ? [...oldItemCurations, ...itemCurations] : itemCurations
+      return {
+        ...state,
+        loading: loadingReducer(state.loading, action),
+        data: {
+          ...state.data,
+          [action.payload.collectionId]: mergedItemCurations
+        },
+        error: null
+      }
+    }
+
+    case PUSH_CHANGES_THIRD_PARTY_ITEMS_SUCCESS: {
+      const { itemCurations } = action.payload
```

Comment:
we can add collectionId here, used below

Reply:
oh yes, totally, updated!